psamm_import/importer.py

```diff
@@ -123,6 +123,33 @@ def _decimal_representer(dumper, data):
     return dumper.represent_scalar('tag:yaml.org,2002:float', value)
 
 
+def default_compartment(model):
+    """If some compounds have no compartment, default compartment is made.
+
+    Returns what the default compartment should be set to.
+    """
```

Comment:
I think the second line is actually a better summary and should go first. Second line could include some more details: "If some compounds have no compartment, a unique compartment name is returned to avoid name collisions". Have a look at PEP 257 for more guidelines on writing docstrings: https://www.python.org/dev/peps/pep-0257/